### DISCUSSION TITLE
Adding github actions and dockerfile

### DISCUSSION THREAD

**Discussion by JuliusDiestra** [OneLoneCoder/olcUTIL_Geometry2D#76] [General]
Hello !
noticed there is not CI checks in the repo and also trying to build locally I missed some dependencies.
Are you interested in : 
1) Github action to build your project.
2) Github action to run your tests.
3) Dockerfile so it is easier for developers to build your project inside a docker container.

Greetings !

Julius

**Comment by Moros1138** [comment]
I agree with points 1 and 2 combined... This is an isolated library that does not depend on anything other than the standard library and as such it would be super easy to set up a test suite and to run that test suite automatically via a github action.

Personally, speaking to point 3, I think a docker container is overkill. Anybody wanting to develop with this should already have a C++ development environment set up and ready, or they might have a project with which they want to integrate this library.

I'd be happy to contribute to the writing of tests if, and/or when, a test framework has been selected and Javid has decided if, and/or when, this is the direction he wants to go with this project.
